tests/AutomationTests/DotNetTests.cs

```diff
@@ -0,0 +1,42 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+/*using Microsoft.Oryx.Automation.Client;
```

Comment:
Are we planning on adding tests before this PR is merged?

Reply:
Yes, working on updating this